WordPress/src/main/res/drawable/ic_tick_vector_blue.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="@color/theme_action_sheet_tick_color"
+    android:viewportHeight="24" android:viewportWidth="24"
```

Comment:
This is more of a project specific request. Since the vector icon has a specific size I think we can add the suffix `_24dp` to align with the other icon resources. Wdyt?